src/plugins/bar-graph/bar-graph.scss

```diff
@@ -0,0 +1,9 @@
+@import "../../components/vars.sass";
+
+.bar-graph-content {
+  width: 100%;
+  // height: 100%;
```

Comment:
Will this commented-out line maybe come into play in future work? Or can it just be removed?

Reply:
Yeah, the content is going to be replaced with something real in the next batch of work.